packages/flame_svg/lib/svg_component.dart

```diff
@@ -7,8 +7,15 @@ import './svg.dart';
 class SvgComponent extends PositionComponent {
   Svg svg;
 
-  SvgComponent.fromSvg(this.svg, {Vector2? position, Vector2? size})
-      : super(position: position, size: size);
+  SvgComponent.fromSvg(
+    this.svg, {
+    Vector2? position,
+    Vector2? size,
+  }) : super(position: position, size: size);
+
+  /// TODO(spydon): Once rc12 is released
```

Comment:
I agree we can change everything to point locally, that would solve my problem with flame_audio as well.

next release we can figure out how to do it, even if manually adjusting versions and paths